hugofs/files/classifier.go

```diff
@@ -93,6 +93,7 @@ const (
 	ContentClassBranch  ContentClass = "branch"
 	ContentClassFile    ContentClass = "zfile" // Sort below
 	ContentClassContent ContentClass = "zcontent"
+	ContentClassZero    ContentClass = "zero"  // Special value for zeroFile
```

Comment:
I don't think we need/want this. A zero file would have an empty ContentClass string, which is the natural zero value, no need to make one up.